Week1/js1-w1-A.A.A.js

```diff
@@ -0,0 +1,175 @@
+// Greetings
```

Comment:
Can you split your homework up in different files? As requested in the assignment:

> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference.